packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -808,6 +808,18 @@ export class AutolinkWindows {
     });
     return changed;
   }
+
+  /** @return The CLI command to invoke autolink-windows independently */
+  public getAutolinkWindowsCommand() {
```

Comment:
nit: consider adding an explicit return type. Can be helpful when skimming through code.